boilingcore/config.go

```diff
@@ -22,6 +22,7 @@ type Config struct {
 	Debug             bool     `toml:"debug,omitempty" json:"debug,omitempty"`
 	AddGlobal         bool     `toml:"add_global,omitempty" json:"add_global,omitempty"`
 	AddPanic          bool     `toml:"add_panic,omitempty" json:"add_panic,omitempty"`
+	SoftDeletes       bool     `toml:"soft_deletes,omitempty" json:"soft_deletes,omitempty"`
```

Comment:
Pedantry: Can we name this `AddSoftDelete` to follow the sort of boolean "Add" or "No" thing we have going on here?